### PR TITLE
Rest improvements importexport

### PR DESCRIPTION
**Pull request by knPerera**
REST APIs on mediation sequences and wsdls

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/TiersApi.java [hunk 1]
```diff
@@ -1,158 +0,0 @@
-package org.wso2.carbon.apimgt.rest.api.publisher;
```
Comment (malinthaprasan) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/TiersApi.java]: Any particular reason to delete this?

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/TiersApiService.java [hunk 2]
```diff
@@ -1,26 +0,0 @@
-package org.wso2.carbon.apimgt.rest.api.publisher;
```
Comment (malinthaprasan) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/TiersApiService.java]: Why was this file deleted?

Comment (knPerera) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/TiersApiService.java]: nope. it is a mistake

**Comment by malinthaprasan** [comment]
Manually merged with #3396 due to conflicts with master branch.
